### ISSUE TITLE
Literal Include Assumes Relative Filepath

### ISSUE THREAD

**Issue by rmorshea** [sphinx-doc/sphinx#5967]
# Summary

Consider sphinx documentation located at the absolute path `/my/project/docs/source`.

Given this, the following literal include statement:

```rst
.. literalinclude:: /my/project/lib/file.py
    :language: python
```

Attempts to include a file at the location:

```
/my/project/docs/source/my/project/lib/file.py
```

Instead of just using the explicit absolute path from the directive:

```
/my/project/lib/file.py
```

# Expected Behavior

`literalinclude` should know that a path beginning with `/` is an absolute path.

# Other Info

* Sphinx == 1.8.3
* Python == 3.6

**Comment by jfbu** [comment]
This matches [our documentation](https://www.sphinx-doc.org/en/1.8/usage/restructuredtext/directives.html#directive-literalinclude) so I am requalifying this issue as a question.

**Comment by rmorshea** [comment]
@jfbu it seems like there should be a way to ask for a truly absolute path. Maybe `//`?

**Comment by rmorshea** [comment]
I've realized that I didn't need the absolute path in the first place:

I was dynamically adding information to a Python docstring which used a `literalinclude` for a json file. However I realized, that I could simply use `code-block`, read the content from the file, and insert that content into the docstring directly.

My scenario was a very niche one, and having `/` default to the path to `docs/source` is much more useful.
